SharedContent/cpp-winui-template/WinUI3TemplateCpp/Scenario1_ShortName.xaml.cpp

```diff
@@ -0,0 +1,46 @@
+﻿#include "pch.h"
+#include "Scenario1_ShortName.xaml.h"
+#if __has_include("Scenario1_ShortName.g.cpp")
+#include "Scenario1_ShortName.g.cpp"
+#endif
+
+using namespace winrt;
+using namespace Microsoft::UI::Xaml;
```

Comment:
Consider adding `using namespace Windows::Foundation;` as well, so you can just say `IInspectable const&`